libraries/botbuilder/tests/teams/messaging-extension-action/src/index.ts

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { config } from 'dotenv';
+import * as path from 'path';
+import * as restify from 'restify';
+
+// Import required bot services.
+// See https://aka.ms/bot-services to learn more about the different parts of a bot.
+import { BotFrameworkAdapter } from 'botbuilder';
+
+// This bot's main dialog.
+import { TeamsActionExtensionBot } from './teamsActionExtensionBot';
```

Comment:
Bot is called ActionBasedMessagingExtensionBot on C# side, shall we keep them consistent naming wise?  Perhaps renamed the C# one?

Reply:
I'll rename this one to ActionBasedMessagingBot, to keep it consistent with the already checked-in scenario in C#. I'm going to add `*.zip` files to the .gitignore in this PR so we don't check in any zipped up manifests with credentials